exo-go/src/process_helpers/index.go

```diff
@@ -34,7 +34,9 @@ func RunAndLog(dir string, env []string, log func(string), commandWords ...strin
 	}
 	process := NewProcess(commandWords...)
 	process.SetDir(dir)
-	process.SetEnv(env)
+	if len(env) > 0 {
+		process.SetEnv(env)
+	}
```

Comment:
I like that, although that should probably be done in a separate PR after your execplus changes have been merged in?